FAQ.md

```diff
@@ -0,0 +1,48 @@
+* How can I deploy a new docker-worker?
+
+First of all, you need proper taskcluster credentials. You can use the
+[taskcluster-cli](https://github.com/taskcluster/taskcluster-cli)
+tool to get it:
+```sh
+# eval $(taskcluster-cli signin --scope <scope needed>)
+```
+You also need the [_secrets_ repo](https://github.com/taskcluster/passwordstore-garbage)
```

Comment:
Let's call this the "Taskcluster team passwordstore repo".

Reply:
Also, I changed the name and edited the README a bit -- https://github.com/taskcluster/passwordstore-readme/blob/master/README.md